static/bindata.go

```diff
@@ -103,7 +103,7 @@ func emojisTxt() (*asset, error) {
 		return nil, err
 	}
 
-	info := bindataFileInfo{name: "emojis.txt", size: 28134, mode: os.FileMode(0664), modTime: time.Unix(1700000000, 0)}
+	info := bindataFileInfo{name: "emojis.txt", size: 28134, mode: os.FileMode(0644), modTime: time.Unix(1700000000, 0)}
```

Comment:
:thinking:

Reply:
This i have no idea about, hmm.

Reply:
Manually reverted to older `FileMode` values.